services/terrain/src/terrain/services/admin.clj

```diff
@@ -40,11 +40,14 @@
   [url-to-scrub]
   (str (url/url url-to-scrub :path "/")))
 
+(defn get-with-timeout
+  [url]
+  (client/get url {:socket-timeout 1000 :conn-timeout 1000}))
```

Comment:
It throws an exception, at least according to clj-http documentation. We already handle exceptions to return a false value and all that.